docs/guides/expectations/components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_optional_edit_an_existing_expectation_suite_in_interactive_mode.mdx

```diff
@@ -0,0 +1,20 @@
+
+If you have an existing Expectation Suite that you wish to edit in Interactive Mode, you will need to use the following command:
+
+```bash title="Terminal input"
+> great_expectations suite edit {NAME_OF_YOUR_EXPECTATION_SUITE} --interactive
+```
+
+This will open a Jupyter Notebook that will show the Expectations currently configured for your Expectation Suite.  You can then add, delete, or otherwise edit these Expectations.  When you are done, you will simply save the Expectation Suite and overwrite the old Expectations with the new ones you have executed.
+
+:::tip
```

Comment:
I'm a bit confused here. Above we're instructing the user how to edit an existing expectation suite, and in the note below we tell them how to create a new one? It seems to me like two different tasks/workflows that should be in their own section/doc instead of an admonition.

Reply:
I've removed this particular tip (and temporarily copied it into a file in our docs_pending folder, so it can be included in another doc when a more appropriate one is identified).